### ISSUE TITLE
can we add a weather forecast as well?

### ISSUE THREAD

**Issue by RMHogervorst** [mozilla-iot/weather-adapter#18]
I believe all providers have a forecast option as well. Or should we create a new app for that?

**Comment by mrstegeman** [comment]
What kind of properties do you think would work best for that? I had considered this but didn’t want to add a ton of extra properties.

**Comment by RMHogervorst** [comment]
I did not think about it all 😅

**Comment by RMHogervorst** [comment]
I was just thinking about usecases, perform an action when it is about to rain for the next 2 hours. I'll look into properties tomorrow

**Comment by mrstegeman** [comment]
It seems to me that the _proper_ way would be to have an array property name `Forecast`. The property would be an array of objects, each having high temp, low temp, rain, etc. Unfortunately, array and object properties are not yet implemented in the gateway. See [here](https://github.com/mozilla-iot/gateway/issues/1383) for more information.

**Comment by RMHogervorst** [comment]
An hack around the proper way would be to add a new device that has weather properties but uses the forecast. A new device that is weather-next-hour or something and has temperature, rain etc.

**Comment by mrstegeman** [comment]
True, but that's just clutter in a different way, i.e. on the dashboard instead of inside the device.
